pkg/test/integration/helper.go

```diff
@@ -244,7 +244,7 @@ func (p *PyroscopeTest) NewRequestBuilder(t *testing.T) *RequestBuilder {
 }
 
 func (p *PyroscopeTest) TempAppName() string {
-	return fmt.Sprintf("pprof.integration.%d",
+	return fmt.Sprintf("pprof-integration-%d",
```

Comment:
The dots were messing with downstream sample type parsing: https://github.com/grafana/pyroscope/blob/8db0333af86d5d8dafa557aea1f9c30535eef4c3/pkg/ingester/pyroscope/ingest_adapter.go#L202-L209